src/ansible_navigator/runner/ansible_doc.py

```diff
@@ -0,0 +1,70 @@
+""" Herewithin lies the ability for ansible-runner
+to run the ansible-doc command
+"""
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
+from typing import Union
+
+
+from ansible_runner import get_plugin_docs
+
+from .base import Base
+
+
+class AnsibleDoc(Base):
+    # pylint: disable=too-many-arguments
+    """abstraction for ansible-doc command-line"""
```

Comment:
What kind of abstraction? Could you make it a full sentence?

Reply:
Fixed in #630